<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RefImageViewer</class>
 <widget class="QMainWindow" name="RefImageViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1251</width>
    <height>919</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Viewer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_5">
    <item row="0" column="0">
     <widget class="QSplitter" name="left_right_splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="vgraphSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QwtPlot" name="v_graphPlot"/>
      </widget>
      <widget class="QSplitter" name="imageSplitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QwtPlot" name="imagePlot"/>
         </item>
         <item row="0" column="1">
          <widget class="QScrollBar" name="imageVerticalScrollBar">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QScrollBar" name="imageHorizontalScrollBar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QwtPlot" name="h_graphPlot"/>
      </widget>
      <widget class="QWidget" name="layoutWidget_grid4">
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="4" column="0">
         <layout class="QGridLayout" name="gridLayout_3"/>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Color Map (zero at center)</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="color_scale"/>
        </item>
        <item row="0" column="0">
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::FieldsStayAtSizeHint</enum>
          </property>
         </layout>
        </item>
        <item row="10" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="9" column="0">
         <widget class="QTableWidget" name="image_table"/>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="table_label">
          <property name="text">
           <string>Image Info</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="1" column="1">
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="peak_max_label">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>X Min</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="x_min_input">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="peak_min_label">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="text">
                   <string>X Max</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="x_max_input">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="back_max_label">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Y Min</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="y_min_input">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="back_min_label">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Y Max</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="y_max_input">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="intensity_label">
            <property name="text">
             <string>Intensity</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <widget class="QSlider" name="intensity_slider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="0" column="0">
             <widget class="QRadioButton" name="radioButton_peakLeft">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Peak Left</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="lineEdit_peakLeft">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(65, 217, 255);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QRadioButton" name="radioButton_backLeft">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Back. Left</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QRadioButton" name="radioButton_backRight">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Back. Right</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QRadioButton" name="radioButton_TOFmin">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="baseSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>TOF min</string>
              </property>
              <property name="checked">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QRadioButton" name="radioButton_TOFmax">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>TOF max</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_peakRight">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(65, 217, 255);</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_backLeft">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 100, 91);</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_backRight">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 100, 91);</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit_TOFmin">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 255, 1);</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QLineEdit" name="lineEdit_TOFmax">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(50, 255, 1);</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QRadioButton" name="radioButton_peakRight">
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="sizeIncrement">
               <size>
                <width>150</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Peak Right</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="horizontalSpacer_7">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Expanding</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>30</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_9"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_11"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1251</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="action_Vscroll"/>
    <addaction name="action_Hscroll"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuColor_Map">
     <property name="title">
      <string>Color Map</string>
     </property>
     <addaction name="actionHeat"/>
     <addaction name="actionGray"/>
     <addaction name="actionNegative_Gray"/>
     <addaction name="actionGreen_Yellow"/>
     <addaction name="actionRainbow"/>
     <addaction name="actionOptimal"/>
     <addaction name="actionMulti"/>
     <addaction name="actionSpectrum"/>
    </widget>
    <widget class="QMenu" name="menuGraph_Selected">
     <property name="title">
      <string>Graph Selected</string>
     </property>
     <addaction name="actionClear_Selections"/>
     <addaction name="actionOverlaid"/>
     <addaction name="actionOffset_Vertically"/>
     <addaction name="actionOffset_Diagonally"/>
    </widget>
    <addaction name="menuColor_Map"/>
    <addaction name="menuGraph_Selected"/>
    <addaction name="actionGraph_Rebinned_Data"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action">
   <property name="text">
    <string>Vertical Scroll Bar</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="action_Hscroll">
   <property name="text">
    <string>Horizontal Scroll Bar</string>
   </property>
  </action>
  <action name="action_Vscroll">
   <property name="text">
    <string>Vertical Scroll Bar</string>
   </property>
  </action>
  <action name="actionVertical_Cut">
   <property name="text">
    <string>Vertical Cut</string>
   </property>
  </action>
  <action name="actionGraph_Rebinned_Data">
   <property name="text">
    <string>Graph Rebinned Data</string>
   </property>
  </action>
  <action name="actionHeat">
   <property name="text">
    <string>Heat</string>
   </property>
  </action>
  <action name="actionGray">
   <property name="text">
    <string>Gray</string>
   </property>
  </action>
  <action name="actionNegative_Gray">
   <property name="text">
    <string>Negative Gray</string>
   </property>
  </action>
  <action name="actionGreen_Yellow">
   <property name="text">
    <string>Green-Yellow</string>
   </property>
  </action>
  <action name="actionRainbow">
   <property name="text">
    <string>Rainbow</string>
   </property>
  </action>
  <action name="actionOptimal">
   <property name="text">
    <string>Optimal</string>
   </property>
  </action>
  <action name="actionMulti">
   <property name="text">
    <string>Multi</string>
   </property>
  </action>
  <action name="actionSpectrum">
   <property name="text">
    <string>Spectrum</string>
   </property>
  </action>
  <action name="actionClear_Selections">
   <property name="text">
    <string>Clear Selections</string>
   </property>
  </action>
  <action name="actionOverlaid">
   <property name="text">
    <string>Overlaid</string>
   </property>
  </action>
  <action name="actionOffset_Vertically">
   <property name="text">
    <string>Offset Vertically</string>
   </property>
  </action>
  <action name="actionOffset_Diagonally">
   <property name="text">
    <string>Offset Diagonally</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
